audio_sync/cli.py

```diff
@@ -216,8 +243,8 @@ def _Main(args):
         args.silence_threshold, args.min_silence_length)
     latencies, dropouts = audio_sync.AnalyzeAudios(
         args.ref_wav_path, args.act_wav_path, settings)
+    max_latency, min_latency, avg_latency = GetStats(latencies)
```

Comment:
This should go inside the `if args.print_stats` below, so that it's executed only upon request.

Reply:
I did it slightly differently by moving CalculatePercentiles() inside "if args.print_percentiles" and keeping GetStats() as is. We need to call one of those to get max_latency and GetStats() is a simpler task.

Reply:
Ah! Had missed the use of max_latency further down. I see why GetStats() needs to stay there.